x-pack/plugins/cloud/public/plugin.tsx

```diff
@@ -147,6 +176,12 @@ export class CloudPlugin implements Plugin<CloudSetup> {
       // In the event of an unexpected error, fail *open*.
       // Cloud admin console will always perform the actual authorization checks.
       .catch(() => setLinks(true));
+
+    return {
+      ContextProvider: ({ children }) => (
```

Comment:
nit: Maybe `CloudContextProvider` since `ContextProvider` is otherwise very generic?

Reply:
Good idea.